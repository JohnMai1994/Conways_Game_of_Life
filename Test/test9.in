new 20
init
6 6 9 6
5 7 5 8 5 9 6 9 7 9 8 9 9 8
-1 -1 
print
step
print
step
print
step
print
step
print
steps 3
print
