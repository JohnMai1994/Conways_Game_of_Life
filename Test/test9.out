____________________
____________________
____________________
____________________
____________________
____________________
______X__X__________
_____X______________
_____X___X__________
_____XXXX___________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
_____XX_____________
____XX_XX___________
_____XXXX___________
______XX____________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____XXXX____________
____X___X___________
____X_______________
_____X__X___________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
_____XX_____________
____XXXX____________
___XX_XX____________
____XX______________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____X__X____________
___X________________
___X___X____________
___XXXX_____________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
___XX_______________
__XXXX______________
_XX_XX______________
__XX________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
